### PR TITLE
DO NOT MERGE.  Verify SonarQube warns us when there's a silly python error

### PR DESCRIPTION
**Pull request by bbockelm**
SonarQube should reject this out-of-hand.

### CODE REVIEW

**Comment by vsoch** [comment]
Maybe try `--reports=n`if there is something weird with the -r and n/y ?

**Comment by bbockelm** [comment]
Actually, it seems like this is an ancient version of `pylint`; going to try the `pip` version instead.

**Comment by vsoch** [comment]
This is fun :)

**Comment by bbockelm** [comment]
You can really rack up the commit count when trying to debug CI builds... and, at 2 minutes per commit, really spend some time.

File: .travis.yml [hunk 1]
```diff
@@ -41,6 +39,8 @@ install:
 
 script:
   - make test
+  - sudo apt-get install python-pip
+  - sudo pip install pylint
   - jdk_switcher use oraclejdk8
   - |-
     PYTHONPATH=libexec/python/ pylint libexec/python/ -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt
```
Comment (vsoch) [inline .travis.yml]: stupid question probably - how/why can you put PYTHONPATH=etc. on the same line without an && or ; to end the line?

Comment (vsoch) [reply .travis.yml]: For example why not like:

      PYTHONPATH=libexec/python/; pylint libexec/python/ -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt" or maybe try just going there like:

or

      cd libexec/python && pylint  -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt"

Comment (bbockelm) [reply .travis.yml]: Oh, that's just the bash syntax for temporarily setting an environment variable over the scope of a single line.

**Comment by vsoch** [comment]
it worked!!!

**Comment by vsoch** [comment]
and my python is terrible, lol

**Comment by vsoch** [comment]
It's going to work this time!!

**Comment by vsoch** [comment]
The pylinting worked! I'm not sure it's hooked up totally with Sonar, but this is something we can figure out.

**Comment by bbockelm** [comment]
Yeah - I want to re-run it in debug mode to make sure it's at least trying to read the file.

**Comment by bbockelm** [comment]
https://codeclimate.com/github/singularityware/singularity/compare/test-python-analysis here's another potential tool for python.  Seems limited to formatting, but at least does that!

**Comment by bbockelm** [comment]
Ok, I think we learned all that is to learn from this.  Closing the PR.

**Pull request closed without merging**